ios-10-3-armv7.cmake

```diff
@@ -26,7 +26,8 @@ include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_common.cmake")
 include(polly_fatal_error)
 
 # Fix try_compile
-set(MACOSX_BUNDLE_GUI_IDENTIFIER com.example)
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_xcode_bundle_identifier.cmake")
```

Comment:
After `polly_common` included we can use `include(pollly_ios_bundle_identifier)`.